modules/activitypub/hack_16834.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
```

Comment:
The hack's name was only briefly mentioned on the forge friend's meet. But what is the exact meaning behind the file name, so that others understand it as well.

Reply:
I renamed the file to `user_settings.go`, since it uses the user settings table introduced in #16834.